src/controllers/api.controller.ts

```diff
@@ -144,17 +144,31 @@ export const claimsGet = async (
 export const claimsFeed = async (
   req: Request,
   res: Response,
-  next: NextFunction
+  next: NextFunction,
 ) => {
   try {
-    // const { search } = req.query; // unused for now, TODO here search
-    const { page = 1, limit = 100, search = "" } = req.query; // defaults provided here
+    let { page = 1, limit = 100, search = "" } = req.query;
+
+    page = parseInt(page.toString());
```

Comment:
we can use `page = +page` here and the other fields instead of using all these methods as it costs somehow

Reply:
The cost difference is nothing. I made it this way to avoid the floating numbers as `+page` will also include decimals